### PR TITLE
now urepo can use custom config that will not be overwritten during u…

### PR DESCRIPTION
**Pull request by kt97679**
…repo update

### CODE REVIEW

**Comment by kt97679** [comment]
@floreo please have a look if you will have chance. This addresses issue we were discussing some time ago.

**Comment by sjafferali** [comment]
👍

File: makefile [hunk 1]
```diff
@@ -22,7 +22,4 @@ pkg: bin
 	    --after-install ../after-install.sh \
 	    --before-remove ../before-remove.sh \
 	    -s dir -t deb -v $(PKG_VERSION) -n $(PKG_NAME) `find . -type f` && \
-	ls | grep -v deb$$ | xargs rm -rf
-
-
-
+	find . ! -name '*.deb' -delete
```
Comment (sjafferali) [inline makefile]: 👍

**Comment by kt97679** [comment]
Thanks a lot for review @sjafferali and others!

File: makefile [hunk 2]
```diff
@@ -22,7 +22,4 @@ pkg: bin
 	    --after-install ../after-install.sh \
 	    --before-remove ../before-remove.sh \
 	    -s dir -t deb -v $(PKG_VERSION) -n $(PKG_NAME) `find . -type f` && \
-	rm -rf `ls|grep -v deb$$`
-
-
-
+	find . ! -name '*.deb' -delete
```
Comment (sjafferali) [inline makefile:25]: 👍

**Pull request merged**